scripts/irods/test/test_iticket.py

```diff
@@ -768,7 +768,7 @@ def test_write_tickets_for_collections_allow_creation_of_data_objects__issue_591
 
             self.admin.run_icommand(['iticket', 'delete', ticket_name])
 
-    def test_tickets_update_catalog_modify_time_when_modified__issue_7125(self):
+    def test_tickets_update_catalog_modify_time_when_modified_admin__issue_7125(self):
```

Comment:
How do you feel about the following?
```
test_ticket_mtime_is_updated_when_ticket_properties_are_modified_by_admin__issue_7125
```

Reply:
This looks clearer that what I had. I will implement it.